src/yogita/ReplaceWordFromStringWithName.java

```diff
@@ -0,0 +1,37 @@
+/*
+ Assignment - 47 : 27th Dec'2020
+Replace second last occurance of "Hi" with your name.
+String msg = "Hi globant Hello Hi Hi globant Hi Hello Hello globant Hi Techno Credits Techno";
+output : Hi globant Hello Hi Hi globant Maulik Hello Hello globant Hi Techno Credits Techno  
+
+ */
+
+package yogita;
+
+public class ReplaceWordFromStringWithName {
+	String replaceWord(String str, String wordReplace) {
+		System.out.println("String is:" + str);
+		String input[] = str.split(" ");
+		int cnt = 0;
+		for (int i = input.length - 1; i >= 0; i--) {
```

Comment:
this also good approach, 2nd approach is you can use substring method, check below code

https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/blob/main/src/brijesh/assignment47/ReplaceSecondLastOccurance.java